server/cmwell-sparql-agent/src/main/scala/cmwell/tools/data/sparql/SparqlProcessorManagerSettings.scala

```diff
@@ -40,6 +40,12 @@ class SparqlProcessorManagerSettings {
     stpSettings.getDuration("cmwell.agents.sparql-triggered-processor.max-delay").toMillis.millis
   val interval: FiniteDuration =
     stpSettings.getDuration("cmwell.agents.sparql-triggered-processor.config-polling-interval").toMillis.millis
+
+  val consumeLengthHint = stpSettings.hasPath("cmwell.agents.sparql-triggered-processor.consumer.fetch-size") match {
```

Comment:
same argument as above - should we want this config to be STP **agent** only?

Reply:
Let's have a dedicated configuration for STP as @e-orz had suggested.

Reply:
Will create stand alone data tools changes in a future PR after discussion with @YoniMataraso